tests/test_skyModel.py

```diff
@@ -72,6 +72,24 @@ def test_nexp_scaling(self):
 
         self.assertNotEqual(image_1.array[0, 0], 0)
         self.assertAlmostEqual(2*image_1.array[0, 0], image_2.array[0, 0])
+     
+    def test_skycounts_function(self):
+        """
+        Test that the sky counts per sec function gives the right result for hte previously 
+        calculated zero points. (Defined as the number of electrons per second for 
+        a 24 magnitude source, the default for skyCountsperSec will be set to u band and 24 mag.)
+        """
+        
+        desc.imsim.read_config()
+    	instcat_file = os.path.join(os.environ['IMSIM_DIR'], 'tests',
```

Comment:
This line has incorrect indentation and should crash your test program.  Can you actually execute this code without it failing?  When I try to run this (after fixing the above errors), I get a `TabError`.